shared/protos/garbage/garbage.proto

```diff
@@ -0,0 +1,234 @@
+syntax = "proto3";
+
+package google.garbage.v1;
+
+import "google/api/annotations.proto";
+import "google/api/client.proto";
+import "google/api/resource.proto";
+import "google/longrunning/operations.proto";
+import "google/protobuf/timestamp.proto";
+
+option (google.api.client_package) = {
+  title: "Testing Garbage"
+  namespace: ["Google", "Cloud"]
+  version: "v1"
```

Comment:
Same. I'd prefer to use something strange that does not look like a version or follow a typical version regex.

Reply:
Trashed it up.